replicated-security/noble-1/proposal-noble-1.json

```
@@ -0,0 +1,19 @@
+{
+    "title": "noble-1 Chain",
+    "description": "# Launching the noble-1 Chain\r\n\r\nThis is the proposal to create the first persistent noble chain, \"noble-1\".\r\n\r\nPlease visit the [cosmos/testnets](https://github.com/cosmos/testnets/tree/master/replicated-security/noble-1/) repo for additional details.",
+    "chain_id": "noble-1",
+    "initial_height": {
+        "revision_height": 1,
+        "revision_number": 1
+    },
+    "genesis_hash": "74f1aa802e267ad0d98b65ed3d34cd271c074e9ff5cbc34ecd5ccfd9f69f2818",
+    "binary_hash": "ed9e62a09b0dbac58e722ccc28b64c9925dfde6efe0c77552ae45b2092cd2dfa",
+    "spawn_time": "2023-02-09T16:00:00.000000Z",
+    "blocks_per_distribution_transmission": 1000,
+    "noble_redistribution_fraction": "0.75",
+    "historical_entries": 10000,
+    "transfer_timeout_period": 1800000000000,
+    "ccv_timeout_period": 259200000000000,
+    "unbonding_period": 86400000000000,
```

```suggestion
    "ccv_timeout_period": 2419200000000000,
    "unbonding_period": 1728000000000000,
```
We adjusted these when we restarted the provider chain. The consumer chains can now use these values:
`CcvTimeoutPeriod`: ~72 hours~ -> 24 days
`UnbondingPeriod`: ~1 day~ -> 20 days